### PR TITLE
Add changesets missing since last release

### PR DESCRIPTION
**Pull request by taras**
## Hey, I just made a Pull Request!

I'm going to add changesets that are missing since the last release. I'm starting with a draft PR so those who're interested can track the progress.

## TODO

This list was generated with `git log b5e6051..HEAD | grep "Merge pull request"`

* [x] #2722 from SDA-SE/feat/async-api-react-update -> https://github.com/spotify/backstage/pull/2724/commits/ac66bb38b0b40b7e5a56b5a40cbf28fa7fc0891d
* [x] #2720 from SDA-SE/feat/correct-readme
* [x] #2716 from spotify/rugvip/nightly-fix
* [x] #2700 from spotify/benjdlambert-patch-2
* [x] #2701 from spotify/rugvip/trouble
* [x] #2698 from spotify/emmaindal-patch-1
* [x] #2674 from lowjoel/postgres-create-database -> https://github.com/spotify/backstage/pull/2724/commits/effd328de7d125d8aad246db38f5a5aa0601f004
* [x] #2695 from SDA-SE/jenkins-docs
* [x] #2687 from spotify/rugvip/nightly
* [x] #2689 from spotify/dependabot/npm_and_yarn/rollup-plugin-typescript2-0.27.3 -> https://github.com/spotify/backstage/pull/2724/commits/e38d40ee302c52b8ae776410bc60ed1355715896
* [x] #2692 from spotify/alund/marketplace/gcp
* [x] #2686 from Marvin9/fix/scaffolder-ssr -> https://github.com/spotify/backstage/pull/2724/commits/67fcced5854d688d5b76a5d93b38b5939e442042
* [x] #2690 from dufcrule/patch-1
* [x] #2683 from spotify/update-roadmap
* [x] #2681 from spotify/freben/test-org
* [x] #2678 from SDA-SE/feat/gauge-storybook
* [x] #2677 from spotify/orkohunter/docker-image-publishing
* [x] #2669 from spotify/freben/github-org -> https://github.com/spotify/backstage/pull/2724/commits/cbb339c278360ca64222fcc28f163c1e66231032
* [x] #2671 from spotify/kat-blog
* [x] #2667 from spotify/timbonicus/fix-api-link
* [x] #2623 from BA1RY/feat/entity-404-page-2266 -> https://github.com/spotify/backstage/pull/2724/commits/f845cecef1abf2aeba6262593406bc1bca3de411
* [x] #2661 from spotify/samiram/fix-banner-pos -> https://github.com/spotify/backstage/pull/2724/commits/f845cecef1abf2aeba6262593406bc1bca3de411
* [x] #2660 from SDA-SE/feat/fix-dark-mode-styling -> https://github.com/spotify/backstage/pull/2724/commits/e99df7625b9f9fc1cb50657488ca22783ab65299
* [x] #2657 from spotify/samiram/add-emptyState -> https://github.com/spotify/backstage/pull/2724/commits/216d91475d69d1ad61f55ff49026cb079b78b262
* [x] #2625 from andrewthauer/feat/codeowners-processor -> https://github.com/spotify/backstage/pull/2724/commits/fa8f2c2bee81ebc19023a9911f52fadb5a45e161
* [x] #2515 from ebarriosjr/cloudbuild-plugin -> https://github.com/spotify/backstage/pull/2724/commits/ff193bab2ef9412e37b416aa0ee6f3d7eceb4e8a
* [x] #2656 from spotify/rugvip/lookup -> https://github.com/spotify/backstage/pull/2724/commits/1687b8fbbf25537a45a1d5072082cc209e0b7ff1
* [x] #2651 from spotify/rugvip/quicker
* [x] #2650 from spotify/orkohunter/docs-techdocs-status-in-faq
* [x] #2647 from spotify/rugvip/changetest
* [x] #2629 from spotify/dependabot/npm_and_yarn/storybook-dark-mode-1.0.3
* [x] #2613 from spotify/rugvip/idlookup
* [x] #2643 from spotify/add-cost-insights-plugin-to-marketplace
* [x] #2543 from mfrinnstrom/azure-scaffolder-prepare
* [x] #2637 from spotify/plugin-cost-insights -> https://github.com/spotify/backstage/pull/2724/commits/cab473771138a9ff543a4febe50750bc03339ce9
* [x] #2641 from spotify/freben/easier-read -> https://github.com/spotify/backstage/pull/2724/commits/81cb943790ae30df3e1e56ddb06cbe466ab3d040
* [x] #2642 from spotify/rugvip/node-cache
* [x] #2630 from spotify/rugvip/node14 -> https://github.com/spotify/backstage/pull/2724/commits/61db1ddc6b9156f9ee1aa776231e197adb1700bc
* [x] #2639 from spotify/blam/esbuild -> https://github.com/spotify/backstage/pull/2724/commits/1d0aec70f035d3c2fad375ee196366e7b18310fa
* [x] #2628 from spotify/freben/entity-placeholder -> https://github.com/spotify/backstage/pull/2724/commits/57d555eb217054ac5d76a0e3db68fbd6b037849a
* [x] #2621 from andrewthauer/fix/circleci-proxy-config
* [x] #2638 from RoadieHQ/mcalus3/travisci-integration
* [x] #2626 from spotify/balestrapatrick/docs-typo
* [x] #2598 from themousepotato/techdocs-progress-bar-fix -> https://github.com/spotify/backstage/pull/2724/commits/8351ad79b00e9fb7b3ed7cc41a89d3f90e2202ba
* [x] #2624 from themousepotato/add-component-ux-fix -> https://github.com/spotify/backstage/pull/2724/commits/fb74f1db60c54f84f2c7c4fb8b89b5c92961e010
* [x] #2616 from spotify/rugvip/noidentity -> https://github.com/spotify/backstage/pull/2724/commits/6f447b3fc5cf9898c5d9cf2fa6913bc3b9ab25e2
* [x] #2614 from spotify/rugvip/internal -> https://github.com/spotify/backstage/pull/2724/commits/d7873e1aa9b4e3afc2de1be414a86b1dd45a42aa
* [x] #2620 from spotify/Rugvip-patch-1
* [x] #2619 from spotify/rugvip/rel
* [x] #2615 from spotify/rugvip/ipvbork -> https://github.com/spotify/backstage/pull/2724/commits/e37c0a0056857241f41e8cc17a860ee7d811f8d6
* [x] #2610 from spotify/rugvip/discovery -> https://github.com/spotify/backstage/pull/2724/commits/30dd111225d5e9aca50e11b9894152fb506bc7cb
* [x] #2611 from spotify/rugvip/api -> https://github.com/spotify/backstage/pull/2724/commits/961414d556447a97d0ba7bf950f69f07f6ea03e4
* [x] #2612 from spotify/rugvip/jankins -> https://github.com/spotify/backstage/pull/2724/commits/1297dcb3ad9bd7d3b7af1e15ccd09bb287ec45fc 
* [x] #2609 from spotify/shmidt-i/fix-create-app-backend -> https://github.com/spotify/backstage/pull/2724/commits/e67d49bf5700645b88cf632a8d6d8ceb3e0d8bf5
* [x] #2535 from spotify/samiram/fix-banner-pos -> https://github.com/spotify/backstage/pull/2724/commits/ae5983387e56d0c98f4e0035063080a5a649f74c
* [x] #2606 from spotify/rugvip/reroute
* [x] #2607 from spotify/rugvip/noproxy
* [x] #2565 from spotify/rugvip/refreg -> https://github.com/spotify/backstage/pull/2724/commits/cbbd271c4d05ae61164a2c43928af422c9668403
* [x] #2600 from spotify/rugvip/disco ->  https://github.com/spotify/backstage/pull/2724/commits/5249594c5fa8fc3b393c72c64db94f75cd55fe20
* [x] #2507 from spotify/dependabot/npm_and_yarn/types/webpack-env-1.15.3
* [x] #2603 from spotify/freben/csp -> https://github.com/spotify/backstage/pull/2724/commits/56e4eb589d71078ed26815e8cb8c37d9710443e2
* [x] #2284 from jibone/add-saml-login -> https://github.com/spotify/backstage/pull/2724/commits/819a70229ad61e20e6853fe133695fc790bfd89a
* [x] #2597 from SDA-SE/feat/api-ingestion -> https://github.com/spotify/backstage/pull/2724/commits/12b5fe940336b6ce4f496fb9946fe79f9e9dcad6
* [x] #2575 from spotify/samiram/fix-dense-table -> https://github.com/spotify/backstage/pull/2724/commits/482b6313d239419f231ad5c4107db0f83fae05e9
* [x] #2530 from timja/use-buster
* [x] #2563 from RoadieHQ/recent-workflows-widget -> https://github.com/spotify/backstage/pull/2724/commits/d67c529abef3c0624e15a3969f7deb78ab7c8aa6
* [x] #2586 from spotify/freben/case-insensitive -> https://github.com/spotify/backstage/pull/2724/commits/e0be86b6faac99ffc9245c03e78d4228506a4fbf
* [x] #2587 from spotify/freben/user -> https://github.com/spotify/backstage/pull/2724/commits/f70a5286853e64a8c915c92ab140413c055e58ac
* [x] #2532 from spotify/freben/ref -> https://github.com/spotify/backstage/pull/2724/commits/3a42365704454387ca3c0d2b10b6e2b4ac31e792
* [x] #2592 from spotify/orkohunter/troubleshooting-techdocs
* [x] #2599 from spotify/orkohunter/codecov-flags-carryforward
* [x] #2541 from spotify/dark-mode-colors -> https://github.com/spotify/backstage/pull/2724/commits/3a42365704454387ca3c0d2b10b6e2b4ac31e792
* [x] #2562 from spotify/rugvip/api
* [x] #1724 from spotify/feat/backend-plugin -> https://github.com/spotify/backstage/pull/2724/commits/28edd7d29c2207e8bab8df487176d9739da4a135
* [x] #2588 from spotify/bug/make-gcp-public
* [x] #9 from spotify/master
* [x] #8 from spotify/master
* [x] #7 from spotify/master
* [x] #6 from spotify/master
* [x] #4 from spotify/master

### CODE REVIEW

File: .changeset/2565.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+---
+'@backstage/core-api': minor
```
Comment (Rugvip) [inline .changeset/2565.md:2]: Should actually be `patch` right?

Comment (taras) [reply .changeset/2565.md]: Done in https://github.com/spotify/backstage/pull/2724/commits/ee1b98030c35cf9a45788342ca89e5197e6f01e6

**Review by Rugvip — APPROVED** [review]
Awesome stuff! :shipit:

**Review by freben — COMMENTED** [review]
totally 🐑

**Review by freben — APPROVED** [review]

**Pull request merged**